src/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  margin-top: 20px;
+}
+
+/* search bar */
+input {
```

Comment:
Font-weight should be 300 by default, 400 for placeholder and 900 for focused style